source/Armament.cpp

```diff
@@ -197,6 +198,26 @@ int Armament::TurretCount() const
 
 
 
+// Determine the installed weaponry's reusable ammunition. That is, all ammo outfits that are not also
+// weapons (as then they would be installed on hardpoints, like the "Nuclear Missile" and other one-shots).
+set<const Outfit *> Armament::RestockableAmmo() const
+{
+	auto restockable = set<const Outfit *>{};
+	for(auto &&hardpoint : Get())
+	{
+		const Weapon *weapon = hardpoint.GetOutfit();
+		if(weapon)
+		{
+			const Outfit *ammo = weapon->Ammo();
+			if(ammo && !ammo->IsWeapon())
```

Comment:
@Amazinite ref 0c4947a1ff9c9ede413dba394f3495b0fc2a8738  -- was that addressing a particular issue?

Reply:
If a weapon has ammo usage, the outfitter attempts to refill as much of that ammo as possible. Normally, the amount of ammunition one can carry is capped by the capacity attribute for that ammo. But if the ammo usage is 0, it's expected that you're not dealing with normal ammunition (i.e. there is no ammo capacity attribute, and the ammo being used likely isn't without outfit space like normal ammo). What was happening was that people were using an ammo usage of 0 where the ammo was some normal outfits (e.g. a jump drive), and then whenever they landed. checked the outfitter, and refilled their ammo, the outfitter would fit as many of that outfit as it possibly could within the limits of the ship's outfit space, therefore cluttering the ship with ammo that wasn't actually needed, since an ammo usage of 0 only requires one instance of the outfit in order to fire.

Reply:
Shouldn't this not include the ammo if the launcher doesn't actually consume it, then?